### DISCUSSION TITLE
how to use rclone

### DISCUSSION THREAD

**Discussion by NekoNis** [tgdrive/teldrive#356] [Q&A]
I made the configuration file as it says on the page but it doesn't work
![image](https://github.com/user-attachments/assets/3bfdc576-eb03-415f-b575-f82171a081f1)
it gives the following erro
![image](https://github.com/user-attachments/assets/fba2aad4-6972-43a2-a81c-b9f52f91221f)

**Comment by Jopo-JP** [comment]
> I made the configuration file as it says on the page but it doesn't work
> ![image] (https://github.com/user-attachments/assets/3bfdc576-eb03-415f-b575-f82171a081f1)
> It gives the following erro.
> ![image] (https://github.com/user-attachments/assets/fba2aad4-6972-43a2-a81c-b9f52f91221f)
> 

You are using the wrong version of rclone. For teldrive, you'll need to take advantage of the special version of rclone because teldrive isn't included in the upstream and will probably never be included. You have to exchange your current rclone file for this one, you can use everything remotely otherwise normally, except that teldrive is also included here. 

https://github.com/tgdrive/rclone
